craft_cli/dispatcher.py

```diff
@@ -525,7 +527,11 @@ def pre_parse_args(
             if self._default_command is None:
                 help_text = self._get_general_help(detailed=False)
                 raise ArgumentParsingError(help_text)
-            emit.trace(f"Using default command: {self._default_command.name!r}")
+            emit.progress(
+                f"Using {self._app_name} without a command will be removed in a future release. "
```

Comment:
```suggestion
                f"Running {self._app_name} without a command will not be possible in future releases."
```

It's subtle, but I'm not jazzed by "using will be removed".

Reply:
https://github.com/canonical/craft-cli/pull/362/commits/04e3ad02886984f54596a8081c9ab9fda964d600